README.md

```diff
@@ -1 +1,15 @@
-# joda-to-java-time-bridge
\ No newline at end of file
+# joda-to-java-time-bridge
+
+[![Build Status](https://travis-ci.org/airlift/joda-to-java-time-bridge.svg?branch=master)](https://travis-ci.org/airlift/joda-to-java-time-bridge)
+
+Mixed use of joda and java.time results in two different versions of
+tzdata being used at the same time. This inconsistencies can often
```

Comment:
I'd use `time zone database` instead of `tzdata`

Reply:
I would remove the word `often`

Reply:
> results in two different zone databases being used at the same time